# Enables or disables the body. If a body is disabled, collision detection
# and physics will will be turned off for it.
string sessionname
int32 sessionid
uint32 bodyid
byte enable
---
